unfoldingWord/translationCore
# Feature bspidel 3222a

#### This pull request addresses:

changes truncation point to be 3 more characters. Adjusts hover text to be wider and lower to fit in card.



#### How to test this pull request:
1 Start tC
2 Select a project
3 Import or view project card for: ab_jhn_reg
4 Verify that language is truncated at end of second line
5 Hover over language
6 Verify that full name is displayed and that tool tip is within Project Card


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3578)
